Sources/AblyChat/Events.swift

```diff
@@ -29,6 +29,34 @@ public enum MessageAction: String, Sendable {
     }
 }
 
+extension MessageAction: InternalRawRepresentable {
+    internal typealias RawValue = String
+
+    internal init?(rawValue: String) {
```

Comment:
Would be nice if we could avoid duplicating the keys. There are various ways you could do this, but perhaps the easiest would be to just have an internal `WireMessageAction: String` type that you convert to and from `MessageAction`.

Reply:
done